### ISSUE TITLE
AttributeError: module 'wandb.proto.wandb_internal_pb2' has no attribute 'Result'

### ISSUE THREAD

**Issue by yangboz** [wandb/wandb#7406]
### Describe the bug

```python

```

```shell
Wandb.login()
```

### Additional Files

import wandb
import random

wandb.login()
#ndb. start a new wandb run to track this script
wandb.init(
    # set the wandb project where this run will be logged
    project="anna-awesome-project",

    # track hyperparameters and run metadata
    config={
    "learning_rate": 0.02,
    "architecture": "CNN",
    "dataset": "CIFAR-100",
    "epochs": 10,
    }
)

# simulate training
epochs = 10
offset = random.random() / 5
for epoch in range(2, epochs):
    acc = 1 - 2 ** -epoch - random.random() / epoch - offset
    loss = 2 ** -epoch + random.random() / epoch + offset

    # log metrics to wandb
    wandb.log({"acc": acc, "loss": loss})

# [optional] finish the wandb run, necessary in notebooks
wandb.finish()

quick tutorial from Wandb website.

### Environment

WandB version:latest

OS:Macosx

Python version:3.9

Versions of relevant libraries: 

### Additional Context

[_No response_](https://docs.wandb.ai/quickstart?_gl=1*f4bqgb*_ga*MTc0OTQ1ODc4Ni4xNzEzNDAyODcy*_ga_JH1SJHJQXJ*MTcxMzQwMjg3MS4xLjEuMTcxMzQwNzQ5Mi42MC4wLjA.)

**Comment by yangboz** [comment]
official quick start  https://docs.wandb.ai/quickstart?_gl=1*f4bqgb*_ga*MTc0OTQ1ODc4Ni4xNzEzNDAyODcy*_ga_JH1SJHJQXJ*MTcxMzQwMjg3MS4xLjEuMTcxMzQwNzQ5Mi42MC4wLjA.

**Comment by SarthakNikhal** [comment]
@kptkin @yangboz Can I help with this issue?

**Comment by kptkin** [comment]
@SarthakNikhal sure go for it :)

**Comment by yangboz** [comment]
> @kptkin @yangboz Can I help with this issue?

sure,  please .

**Comment by SarthakNikhal** [comment]
@yangboz I'm already on it and almost done. I'll let you know if I need any help.

**Comment by SarthakNikhal** [comment]
@yangboz Hey. I tried running the example above with the same settings and it worked for me. You can try updating your python version if it does not conflict with your work and let me know

**Comment by astroesteban** [comment]
I was running into the same bug in my Python 3.12 virtual environment. I found that the following combination worked for me:
```
protobuf==3.20
wandb==0.16.6
```

**Comment by thanos-wandb** [comment]
Hi @yangboz thank you for reporting this issue. Could you please let us know your current `wandb` and `protobuf` version? would it work for you when upgrading to the versions suggested from @astroesteban?

**Comment by rumourscape** [comment]
> I was running into the same bug in my Python 3.12 virtual environment. I found that the following combination worked for me:
> 
> ```
> protobuf==3.20
> wandb==0.16.6
> ```

This works :+1:

**Comment by richardrl** [comment]
Still see this error with protobuf 5.26.1 and wandb 0.16.6

**Comment by thanos-wandb** [comment]
Hi @richardrl could you please try to update `protobuf` to a version `<5`? Here are our dependencies of your current `wandb v0.16.6`: [https://github.com/wandb/wandb/blob/v0.16.6/pyproject.toml](https://github.com/wandb/wandb/blob/v0.16.6/pyproject.toml)

**Comment by BenDes21** [comment]
Same issue here, did all the recommendation's but still the message : AttributeError: module 'wandb.proto.wandb_internal_pb2' has no attribute 'Result'

**Comment by thanos-wandb** [comment]
Hi @BenDes21 may I please ask what's your current wandb and protobuf versions?

**Comment by gerardsimons** [comment]
I just upgraded to the latest version of wandb (pip install -U wandb) and that fixed the issue for me on Python 3.9 for Mac.

**Comment by tianqitang1** [comment]
No luck for python 3.12:( Had protobuf 4.25.3, there is no protobuf 3 available for python 3.12

**Comment by Darius-H** [comment]
> I was running into the same bug in my Python 3.12 virtual environment. I found that the following combination worked for me:
> 
> ```
> protobuf==3.20
> wandb==0.16.6
> ```

I solved this problem with
```
wandb==0.17.5
protobuf==4.25.3
```
and my python version is Python 3.9.19
